lib/internal/util/inspect.js

```diff
@@ -660,12 +657,10 @@ function getPrefix(constructor, tag, fallback, size = '') {
 // Look up the keys of the object.
 function getKeys(value, showHidden) {
   let keys;
-  const symbols = ObjectGetOwnPropertySymbols(value);
   if (showHidden) {
-    keys = ObjectGetOwnPropertyNames(value);
-    if (symbols.length !== 0)
-      keys.push(...symbols);
+    keys = ReflectOwnKeys(value);
   } else {
+    const symbols = ObjectGetOwnPropertySymbols(value);
     // This might throw if `value` is a Module Namespace Object from an
     // unevaluated module, but we don't want to perform the actual type
     // check because it's expensive.
```

Comment:
I haven't tested the performance of this, but I think this whole function body could be replaced by:

```suggestion
  return showHidden ? ReflectOwnKeys(value) :
                      ArrayPrototypeFilter(
                        ReflectOwnKeys(value),
                        (key) => typeof key === 'string' ||
                                 ObjectPrototypePropertyIsEnumerable(value, key)
                      );
```

Reply:
This would definitely be slower. It creates a new array and it runs a typeof check on each entry besides symbols. That's both overhead that is explicitly not there in the current implementation.